vnext/local-cli/generator-windows/templates/cs/src/App.xaml.cs

```diff
@@ -31,5 +32,23 @@ public App()
 
             InitializeComponent();
         }
+
+        /// <summary>
+        /// Invoked when the application is launched normally by the end user.  Other entry points
+        /// will be used such as when the application is launched to open a specific file.
+        /// </summary>
+        /// <param name="e">Details about the launch request and process.</param>
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
```

Comment:
>OnLaunched [](start = 32, length = 10)

Could we make the C++ and C# code match each other?
Here we check if frame is empty, but we do not do it in the C++ case.